huaweicloud/services/sdrs/resource_huaweicloud_sdrs_replication_pair.go

```diff
@@ -0,0 +1,198 @@
+package sdrs
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/chnsz/golangsdk/openstack/sdrs/v1/replications"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+)
+
+func ResourceSDRSReplicationPair() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceSDRSReplicationPairCreate,
+		ReadContext:   resourceSDRSReplicationPairRead,
+		UpdateContext: resourceSDRSReplicationPairUpdate,
+		DeleteContext: resourceSDRSReplicationPairDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(10 * time.Minute),
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"group_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"volume_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"delete_target_volume": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				ForceNew: true,
+			},
```

Comment:
```suggestion
			"delete_target_volume": {
				Type:     schema.TypeBool,
				Optional: true,
			},
```
Changing this does not create a new resource.